pointerevents/pointerevent_capture_suppressing_mouse-manual.html

```diff
@@ -8,6 +8,7 @@
         <script src="/resources/testharnessreport.js"></script>
         <!-- Additional helper script for common checks across event types -->
         <script type="text/javascript" src="pointerevent_support.js"></script>
+        <script type="text/javascript" src="../dist/pep.js"></script>
```

Comment:
We can't modify this file. The change needs to be submitted against https://github.com/w3c/web-platform-tests/